web/cypress/integration/fire-behaviour-advisory-calculator-page.spec.ts

```diff
@@ -2,6 +2,30 @@ import { FIRE_BEHAVIOR_CALC_ROUTE } from '../../src/utils/constants'
 import { FuelTypes } from '../../src/features/fbaCalculator/fuelTypes'
 import { DateTime } from 'luxon'
 
+const datePickerTest = (commandToUse: Function, visitAndAddRow: Function) => {
+  cy.intercept('GET', 'api/stations/*', { fixture: 'weather-stations.json' }).as('getStations')
+
+  const yesterday = DateTime.now().minus({ days: 1 }).toISODate().slice(0, 10) // 'YYYY-MM-DD'
+
+  cy.intercept('POST', 'api/fba-calc/stations', req => {
+    expect(req.body).to.deep.include({
+      date: yesterday
+    })
+  }).as('calculateResults')
+
+  visitAndAddRow()
+
+  cy.wait('@getStations')
+
+  commandToUse(yesterday)
```

Comment:
```suggestion
  setDate(yesterday)
```

Just think it would read easier this way?